src/main/kotlin/no/nav/tpts/mottak/App.kt

```diff
@@ -10,22 +13,46 @@ import io.ktor.server.netty.*
 import io.prometheus.client.CollectorRegistry
 import io.prometheus.client.exporter.common.TextFormat
 import mu.KotlinLogging
+import no.nav.tpts.mottak.applications.applicationRoutes
+import java.net.URI
 
-private val LOG = KotlinLogging.logger {}
+val LOG = KotlinLogging.logger {}
 
 fun main() {
     LOG.info { "starting server" }
+
+    val issuer = System.getenv("AZURE_ISSUER")
```

Comment:
Issuer is the well-known endpoint without the `/.well-known/openid-connect` suffix